lua/lvim/core/cmp.lua

```diff
@@ -118,14 +118,15 @@ end
 M.methods.jumpable = jumpable
 
 M.config = function()
-  local status_cmp_ok, cmp = pcall(require, "cmp")
+  local status_cmp_ok, cmp_types = pcall(require, "cmp.types.cmp")
```

Comment:
wouldn't this require `cmp` anyway? I think we would need to get rid of this condition

Reply:
no, cmp is not required with this, I checked with --startuptime

Reply:
we don't need it anyways, so let's remove this artificial guard.